tests/components/recollect_waste/test_config_flow.py

```diff
@@ -12,61 +13,69 @@
 from homeassistant.config_entries import SOURCE_USER
 from homeassistant.const import CONF_FRIENDLY_NAME
 
+from .conftest import TEST_PLACE_ID, TEST_SERVICE_ID
 
-async def test_duplicate_error(hass, config, config_entry):
-    """Test that errors are shown when duplicates are added."""
+
+@pytest.mark.parametrize(
```

Comment:
Seems like this has only one parameter, and these params are only used in this test and not any other fixtures. Is that right? If so, then i think this can be inlined?

Reply:
That's correct—that said, I'm wanting to do more error handling in a subsequent PR, and if this test is parameterized, it'll be a trivial add.

Reply:
Great, makes sense if you're planning to add more here in a subsequent PR.